vector/v.net.bridge/v.net.bridge.html

```diff
@@ -43,8 +43,3 @@ <h2>AUTHORS</h2>
 
 Daniel Bundala, Google Summer of Code 2009, Student<br>
 Wolf Bergenheim, Mentor
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
`<h2>EXAMPLES</h2>` should be `<h2>EXAMPLE</h2>`

`<a href="v.category.html">v.category</a>,` should be `<a href="v.category.html">v.category</a>`